integration-tests/src/test/groovy/com/okta/sdk/tests/it/util/ITSupport.groovy

```diff
@@ -77,4 +86,49 @@ abstract class ITSupport implements ClientProvider {
 
         return user
     }
+
+    Group randomGroup(String name = "group-${UUID.randomUUID().toString()}") {
```

Comment:
I think we talked about this before, but don't you have to include an SDK prefix in your random names? Something like "java-sdk"?

Reply:
+1, i'll try to add this, it doesn't work for all fields, as some of them have a shorter name description, but it's easy to try!

Reply:
groups worked, policy's were limited to 50 chars, but i did add `-java-` in the name